plugins/welcome-plugin/package.json

```diff
@@ -11,6 +11,19 @@
     "src",
     "resources"
   ],
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "Welcome plugin configuration",
+      "properties": {
+        "che.welcome.plugin.enable": {
```

Comment:
IMHO the dot looks strange between welcome and plugin

Reply:
@benoitf So, maybe simple `che.welcome.enable` or `che.welcomePlugin.enable`?